libs/api/domains/auth-admin/src/lib/client/clients.service.ts

```diff
@@ -313,30 +312,21 @@ export class ClientsService extends MultiEnvironmentService {
     })
   }
 
-  async deleteClient(
-    user: User,
-    input: DeleteClientInput,
-  ): Promise<DeleteClientResponse[]> {
-    const response = input.environments.map((environment) => ({
-      environment,
-      success: true,
-    }))
-
+  async deleteClient(user: User, input: DeleteClientInput): Promise<boolean> {
     await Promise.all(
-      response.map((resp, index) =>
-        this.adminApiByEnvironmentWithAuth(resp.environment, user)
+      Object.values(Environment).map((resp) =>
+        this.adminApiByEnvironmentWithAuth(resp, user)
```

Comment:
We have a constant array of environment which you can just use directly instead of `Object.values`
```suggestion
      environments.map((env) =>
        this.adminApiByEnvironmentWithAuth(env, user)
```